jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2256,17 +2256,38 @@ public boolean hasFullPrivilegeAccess() {
 		}
 
 		static class ClassDefiner {
-			Class<?> defineClass(boolean option) {
-				return null;
+			private byte[] classBytes;
+			private String className;
+			private Lookup lookup;
```

Comment:
The class and all its fields should be `final`.

Reply:
Fixed.